microsoftgraph/microsoft-graph-explorer
# Task/build versioning

## Overview

Creates a versioned build file ie `explorer_v1.7.8.js`

### Demo

N/A

### Notes

N/A

## Testing Instructions

* Run `npm run build:prod`
* Confirm that a new file has been created in the dist folder `explorer_v1.7.8.js`
